src/Symfony/Component/HttpFoundation/JsonResponse.php

```diff
@@ -131,6 +131,10 @@ public function setData($data = array())
                     restore_error_handler();
                 }
             } catch (\Exception $e) {
+            } catch (\Throwable $e) {
```

Comment:
Useless as the logic inside the catch statement only has extra logic in 2 cases before rethrowing:

- on PHP 5.4 and older, where we will never get a Throwable
- when the exception class is ``Exception`` (and extra condition), which will always be catched by the ``} catch (\Exception $e) {``